projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -1073,6 +1086,27 @@ export class IgxGridBaseDirective extends DisplayDensityBase implements
         this._filteringStrategy = classRef;
     }
 
+    /**
+     * Gets the sorting strategy of the grid.
+     * ```typescript
+     *  let sortStrategy = this.grid.sortStrategy
+     * ```
+     */
+    @Input()
+    get sortStrategy(): IGridSortingStrategy {
+        return this._sortingStrategy;
+    }
+
+    /**
+     * Sets the sorting strategy of the grid.
+     * ```html
+     *  <igx-grid #grid [data]="localData" [sortStrategy]="sortStrategy"></igx-grid>
+     * ```
+     */
+    set sortStrategy(classRef: IGridSortingStrategy) {
```

Comment:
I don't think this is a class reference. It's instance, isn't it?

Reply:
It is an instance and it has been updated.